submarine-sdk/pysubmarine/example/submarine_job_sdk.ipynb

```diff
@@ -75,25 +75,22 @@
    },
    "outputs": [],
    "source": [
-    "jobLibrarySpec = JobLibrarySpec(name='tensorflow',\n",
-    "                                version='2.1.0',\n",
-    "                                image='gcr.io/kubeflow-ci/tf-mnist-with-summaries:1.0',\n",
+    "experiment_meta = ExperimentMeta(name='mnist',\n",
+    "                                namespace='submarine',\n",
+    "                                framework='Tensorflow',\n",
     "                                cmd='python /var/tf_mnist/mnist_with_summaries.py'\n",
     "                                    ' --log_dir=/train/log --learning_rate=0.01'\n",
     "                                    ' --batch_size=150',\n",
     "                                env_vars={'ENV1': 'ENV1'})\n",
     "\n",
-    "worker = JobTaskSpec(name='tensorlfow',\n",
-    "                     image=None,\n",
-    "                     cmd=None,\n",
-    "                     env_vars=None,\n",
-    "                     resources='cpu=4,memory=2048M',\n",
-    "                     replicas=1)\n",
+    "worker_spec = ExperimentTaskSpec(resources='cpu=4,memory=2048M',\n",
+    "                                replicas=1)\n",
+    "\n",
+    "environment = Environment(image='gcr.io/kubeflow-ci/tf-mnist-with-summaries:1.0')\n",
     "\n",
-    "body = JobSpec(name='mnist',\n",
-    "               namespace='submarine',\n",
-    "               library_spec=jobLibrarySpec,\n",
-    "               task_specs={\"Worker\": worker})"
+    "experiment_spec = ExperimentSpec(meta=experiment_meta,\n",
+    "                                 environment=environment,\n",
+    "                                 spec={\"Worker\": worker_spec}) \n"
    ]
   },
   {
```

Comment:
The response is outdated and needs an update.
```
"            'spec': {'librarySpec': {'cmd': 'python '\n",
       "                                            '/var/tf_mnist/mnist_with_summaries.py '\n",
       "                                            '--log_dir=/train/log '\n",
       "                                            '--learning_rate=0.01 '\n",
       "                                            '--batch_size=150',\n",
       "                                     'envVars': {'ENV1': 'ENV1'},\n",
       "                                     'image': 'gcr.io/kubeflow-ci/tf-mnist-with-summaries:1.0',\n",
       "                                     'name': 'tensorflow',\n",
       "                                     'version': '2.1.0'},\n",
```

Reply:
I will refactor client API in the next Jira.  The next PR will modify the output again.
so I didn't update the output temporarily.
I could update it if needed.

Reply:
Thanks, I've updated it